modApiServer/src/org/aion/api/server/zmq/ProtocolProcessor.java

```diff
@@ -144,6 +186,39 @@ public void run() {
         }
     }
 
+    private void loadCurveKeyPair() {
+        List<File> files = getFiles();
+        String nextLoad = "";
+        for (File f : files) {
```

Comment:
Is there a benefit to this fall-back/fuzzy logic?  Why not just have two hard-coded file names it looks for?

Reply:
I was thinking the user will generate several keypairs in case need to switch the keyset. The public key also needs to be updated to the client side, then the connection will be setup.